index.html

```diff
@@ -0,0 +1,187 @@
+                <p class="article__text">Trendy and fresh design, fits any website.</p>
+            </article>
+        </div>
+
+    </section>
+
+    <section class="logo-bar logo">
+
+        <div class="logo__list">
+            <img  class="logo__img" src="./images/logos.png" alt="List of partners logos">
+        </div>
+
+    </section>
+
+    <section class="container">
+
+        <div class="container__img-box">
+            <img class="container__img" src="./images/imac.png" alt="iMac">
+        </div>
+        
+        <aside class="container__text-box">
+            <h3 class="text-box__title">SIMPLE AND BEAUTIFUL</h3>
+            <p class="text-box__paragraph">Use Nova theme for your next web project. It is completely customizable so you can change any of the sections to fit your needs. Nova Theme is Free for any kind of use, personal and commercial. Have fun and good luck!</p>
+        </aside>
+
+    </section>
+
+    <section class="container--beige">
+
+        <h3 class="container__title">CHOOSE YOUR PRICING PLAN</h3>
+        <p class="container__subtitle">Pick any of our super affordable pricing plans</p>
+        
+        <div class="container__container">
+
+            <div class="container__article article">
+                <h4 class="article__head">BASIC</h4>
+                <h2 class="article__prize">$15</h2>
+                <p class="article__paragraph--small">per month</p>
+                <p class="article__paragraph--normal">
+                    Up to 7 Projects<br />
+                    2 Additional Developers
+                </p>
+                <div class="article__button"><a class="article__link" href="">Get Started</a></div>
+            </div>
+
+            <div class="container__article article">
+                <h4 class="article__head">AGENCY</h4>
+                <h2 class="article__prize">$55</h2>
+                <p class="article__paragraph--small">per month</p>
+                <p class="article__paragraph--normal">
+                    Up to 7 Projects<br />
+                    2 Additional Developers<br />
+                    Unlimited Support
+                </p>
+                <div class="article__button"><a class="article__link" href="">Get Started</a></div>
+            </div>
+
+            <div class="container__article article">
+                <h4 class="article__head">PRO</h4>
+                <h2 class="article__prize">$75</h2>
+                <p class="article__paragraph--small">per month</p>
+                <p class="article__paragraph--normal">
+                    Up to 7 Projects<br />
+                    2 Additional Developers<br />
+                    Unlimited Support<br />
+                    1.5GB Disk Space
+                </p>
+                <div class="article__button"><a class="article__link" href="">Get Started</a></div>
+            </div>
+
+        </div>
+    </section>
+
+    <section class="container--white">
```

Comment:
Nigdy modyfikator (czy to do blocku czy elementu z BEM) nie występuje samodzielnie czyli powinno być `container container--white`.
Więcej znajdziesz tutaj: https://devmentor.pl/b/metodologia-bem-w-css-i-sass